engine/userguide/networking/default_network/build-bridges.md

```diff
@@ -11,70 +11,68 @@ install Docker.
 > **Note**: The [Docker networks feature](../index.md) allows you to
 create user-defined networks in addition to the default bridge network.
 
-You can set up your own bridge before starting Docker and use `-b BRIDGE` or
-`--bridge=BRIDGE` to tell Docker to use your bridge instead. If you already
-have Docker up and running with its default `docker0` still configured,
-you can directly create your bridge and restart Docker with it or want to begin by
-stopping the service and removing the interface:
+You can set up your own bridge before starting Docker and configure Docker to
+use your bridge instead of the default `docker0` bridge.
 
-```
-# Stopping Docker and removing docker0
+1.  Configure the new bridge.
 
-$ sudo service docker stop
+    ```bash
+    $ sudo brctl addbr bridge0
 
-$ sudo ip link set dev docker0 down
+    $ sudo ip addr add 192.168.5.1/24 dev bridge0
 
-$ sudo brctl delbr docker0
+    $ sudo ip link set dev bridge0 up
+    ```
 
-$ sudo iptables -t nat -F POSTROUTING
-```
+    Confirm the new bridge's settings.
 
-Then, before starting the Docker service, create your own bridge and give it
-whatever configuration you want. Here we will create a simple enough bridge
-that we really could just have used the options in the previous section to
-customize `docker0`, but it will be enough to illustrate the technique.
+    ```bash
+    $ ip addr show bridge0
 
-```
-# Create our own bridge
+    4: bridge0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state UP group default
+        link/ether 66:38:d0:0d:76:18 brd ff:ff:ff:ff:ff:ff
+        inet 192.168.5.1/24 scope global bridge0
+           valid_lft forever preferred_lft forever
+    ```
 
-$ sudo brctl addbr bridge0
+2.  Configure Docker to use the new bridge by setting the option in the
+    `daemon.json` file, whose default location is `/etc/docker/daemon.json` on
+    Linux. If the `daemon.json` file does not exist, create it. Assuming there
+    are no other settings in the file, it should have the following contents:
 
-$ sudo ip addr add 192.168.5.1/24 dev bridge0
+    ```json
+    {
+      "bridge" : "bridge0"
```

Comment:
Stylistically, there generally isn't a space between the key and the `:` character.

```
"bridge": "bridge0"
```